src/core/public/mocks.ts

```diff
@@ -30,6 +30,7 @@ import { themeServiceMock } from './theme/theme_service.mock';
 export { chromeServiceMock } from './chrome/chrome_service.mock';
 export { docLinksServiceMock } from './doc_links/doc_links_service.mock';
 import { executionContextServiceMock } from './execution_context/execution_context_service.mock';
+export { executionContextServiceMock } from './execution_context/execution_context_service.mock';
```

Comment:
Outside of the scope of the PR, but would you mind moving the `import` statement with the others a few lines up 🙏 ?

Reply:
👍 done